src/templates/article.js

```diff
@@ -179,11 +179,17 @@ const Article = props => {
         meta.updatedDate,
         dateFormatOptions
     );
+    const canonicalUrl = dlv(
+        __refDocMapping,
+        'ast.options.canonical-href',
+        null
```

Comment:
```suggestion
        og.url || articleUrl
```
We can actually set this to the fallback case for `dlv` and then remove the conditional in `SEO` since we can handle it here!